src/legacy/core_plugins/dashboard_embeddable/public/embeddable/_variables.scss

```
@@ -0,0 +1 @@
+$dshEditingModeHoverColor: transparentize($euiColorWarning, lightOrDarkTheme(.9, .7));
```

Change prefix to `emb`
```suggestion
$embEditingModeHoverColor: transparentize($euiColorWarning, lightOrDarkTheme(.9, .7));
```